go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/prometheus/client_model v0.5.0
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.8.4
```

Comment:
This package is currently not expected to be imported

Reply:
Okay , I will refactor the code without using this package.